src/PyHyperScattering/SST1RSoXSDB.py

```diff
@@ -1025,72 +1045,46 @@ def loadMd(self, run):
             primary = run["primary"]["data"]
         except (KeyError, HTTPStatusError):
             raise Exception(
-                "No primary stream --> probably you caught run before image was written.  Try"
-                " again."
+                "No primary stream --> probably you caught run before image was written.  Try again."
             )
 
         md_lookup = copy.deepcopy(self.md_lookup)
-        md_secondary_lookup = copy.deepcopy(self.md_secondary_lookup)
-
+        ## Add additional metadata not included in lookup dictionary
+        md_key_names_beamline = []
+        for key in md_lookup.keys(): md_key_names_beamline = md_key_names_beamline + md_lookup[key] ## Making single list with all historical keys to check if they are in primary
```

Comment:
```suggestion
        for key in md_lookup.keys(): ## Make a single list with all historical keys to check if they are in primary
            md_key_names_beamline = md_key_names_beamline + md_lookup[key] 
```
more readable to not have it on one line.